roles/system/tasks/tools.yml

```diff
@@ -18,7 +18,7 @@
       - libxslt1-dev
       - libffi-dev
       - curl
-      - libcurl3
+      - libcurl4
```

Comment:
Unfortunately this change breaks installations on Ubuntu 16.04. Maybe we could remove this line and assume the other dependencies likue `curl` and `libcurl4-openssl-dev` will depend on the right `libcurl` package :thinking:.

Reply:
Sure! I'll look for other 18.04 related issues and I'll try to fix it.

Reply:
Hi, @javimoral :smile:.

Are you still working on this pull request?

Reply:
Sorry! I've been a little busy. I'll work on it this weekend and submit a patch by Monday.

Reply:
@javimoral How are you doing?

Are you still working on this pull request? If not, I can take it from here :wink:.

Reply:
Sorry for the long delay but it's being quite a rough days, so please, yes, continue by yourself.

Sorry and keep yourself safe!

Reply:
No problem :smile:. Cheers!